CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 - Update asab-webui submodule (INDIGO Sprint 230512, [!31](https://github.com/TeskaLabs/seacat-auth-webui/pull/31))
```

Comment:
@Aringoaway Those submodule updates mentioned separately does not have to be here, it does not have any value for person reading the changelog